Makefile

```diff
@@ -1,15 +1,16 @@
 SHELL := /bin/bash
 GO := GO15VENDOREXPERIMENT=1 go
+GOPATH := $(shell go env GOPATH)
```

Comment:
I think this is going to be clobbered by the [buildpack](https://github.com/jenkins-x-buildpacks/jenkins-x-classic/blob/6ce2932ce0e30d4fbe98c0c72b76152cd7393e8b/packs/go/Makefile#L2).

Reply:
I think this is fine if it's over written by the buildpack. It was added mainly to guarantee a value is set for GOPATH when running locally.